api/spec/controllers/mno_enterprise/jpi/v1/admin/organizations_controller_spec.rb

```diff
@@ -60,12 +64,64 @@ module MnoEnterprise
         it { subject; expect(data['organizations'].first['id']).to eq(organization.id) }
         it_behaves_like 'an unauthorized route for support users'
       end
+    end
+
+    describe 'GET #support_search' do
+      subject { get :support_search, params}
+      let(:admin_role) { :support }
+      let(:params) { {} }
+      let(:data) { JSON.parse(response.body) }
+
+      context 'with an invalid search' do
+        it_behaves_like 'an unauthorized route for support users'
+        let(:params) do
+          {
+            org_search: {
+              where: {
+                'name.like' => 'a'
+              }
+            }.to_json,
+            user_search: {
+              where: {
+                'name.like' => 'a',
+                'surname.like' => 'a'
+              }
+            }.to_json
+          }
+        end
+
+        it_behaves_like 'an unauthorized route for support users'
+
+        let(:params) do
+          {
+            org_search: {
+              where: {
+                admin_role: 'Admin'
+              }
+            }.to_json,
+            user_search: {
+              where: {
+                admin_role: 'Admin'
```

Comment:
Probably does not matter too much, but should this be lowercased to `admin`? @adamaziz15 and I were wondering about this.

Reply:
Eh the string doesn't matter. This spec is mainly to check that you cannot search by an unpermitted search parameter.